frontends/web/src/routes/new-settings/advanced-settings.tsx

```diff
@@ -0,0 +1,45 @@
+import { useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useLoad } from '../../hooks/api';
+import { Main, Header } from '../../components/layout';
+import { View, ViewContent } from '../../components/view/view';
+import { WithSettingsTabs } from './components/tabs';
+import { TPagePropsWithSettingsTabs } from './type';
+import { EnableCustomFeesToggleSetting } from './components/advanced-settings/enable-custom-fees-toggle-setting';
+import { ConnectFullNodeSetting } from './components/advanced-settings/connect-full-node-setting';
+import { EnableTorProxySetting } from './components/advanced-settings/enable-tor-proxy-setting';
+import { getConfig } from '../../api/backend';
+import InlineMessage from '../../components/inlineMessage/InlineMessage';
+
+export const AdvancedSettings = ({ deviceIDs, hasAccounts }: TPagePropsWithSettingsTabs) => {
+  const { t } = useTranslation();
+  const fetchedConfig = useLoad(getConfig);
+  const [config, setConfig] = useState<any>();
+  const [showRestartMessage, setShowRestartMessage] = useState(false);
+
+  useEffect(() => {
+    setConfig(fetchedConfig);
+  }, [fetchedConfig]);
```

Comment:
Fetching the config here **once** since it's being used in two components in the page (instead of calling the same exact thing twice from 2 diff setting item components). 

Ive been trying to minimise states at the page-level and just have them inside of each setting item to have a neater code, but IMO this one makes sense to have it here.

Reply:
not important rightnow but what do you think of, instead of passing the whole config to the components, only pass the relevant part?

also I am wondering if it is really needed to receive updates via onChangeConfig and pass config down again... but no blocker. keep as is or feel free to change.